nestkernel/conn_builder.cpp

```diff
@@ -74,75 +71,145 @@ nest::ConnBuilder::ConnBuilder( NodeCollectionPTR sources,
   updateValue< bool >( conn_spec, names::make_symmetric, make_symmetric_ );
 
   // read out synapse-related parameters ----------------------
-  if ( not syn_spec->known( names::synapse_model ) )
-  {
-    throw BadProperty( "Synapse spec must contain synapse model." );
-  }
-  const std::string syn_name = ( *syn_spec )[ names::synapse_model ];
-  if ( not kernel().model_manager.get_synapsedict()->known( syn_name ) )
+
+  // synapse-specific parameters
+  std::set< Name > skip_set = {
```

Comment:
The purpose of the `skip_set` is not clear, and neither the comment nor the name help much. Can we find  a good name to express that these are parameters that are either not parameters for individual synapses or are handled directly? Or maybe just comment it ...